### DISCUSSION TITLE
Caprover Non Root User

### DISCUSSION THREAD

**Discussion by rdiok3** [caprover/caprover#2003] [Q&A]
Dear Community,

I have been testing Caprover and noted 
that it executes processes as root user in Ubuntu.
(was testing the hard drive access from Django).

Do you know how can I set Caprover to use a
non'root user ?.  For security reasons I have 
a separate admin (non-root) user with sudo rights.

Thank you very much for your help

**Comment by githubsaturn** [comment]
Are you referring to the container user? You can always set it manually.

https://www.geeksforgeeks.org/docker-user-instruction/
